python/paddle/fluid/transpiler/inference_transpiler.py

```diff
@@ -20,12 +21,12 @@
 
 class InferenceTranspiler:
     '''
-    Convert the fluid program to optimized inference program. 
-    
-    There are several optimizations, only fuse batch normalization is supported now.
+    Convert the fluid program to optimized inference program.
+
+    There are several optimizations.
```

Comment:
There are several optimizations:

- fuse convolution and batch normalization
- fuse batch normalization and relu (MKLDNN only)